### PR TITLE
Fix manifest version in oldstable branch

### PR DESCRIPTION
**Pull request by rodinux**
correct version 27.1.6 for this oldstable

## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by rodinux** [comment]
!testme

File: conf/nginx.conf [hunk 1]
```diff
@@ -2,10 +2,10 @@
...
 # or include you can include that list explicitly and add the file extension
 # only for Nextcloud like below:
-include mime.types;
-types {
-    text/javascript js mjs;
-}
+#include mime.types;
+#types {
+#    text/javascript js mjs;
+#}
```
Comment (OniriCorpe) [inline conf/nginx.conf:8]: hi!
why have you commented this?

Comment (rodinux) [reply conf/nginx.conf:8]: These lines was not here on v27.1.4, they have been added after a bug with the v28...
https://github.com/nextcloud/server/issues/42989

Comment (OniriCorpe) [reply conf/nginx.conf:8]: oh, i see, thanks!

Comment (rodinux) [reply conf/nginx.conf:8]: And also there is a bug https://github.com/nextcloud/server/issues/42989

Comment (rodinux) [reply conf/nginx.conf:8]: I have to be careful with links...

Comment (rodinux) [reply conf/nginx.conf:8]: I think this line is enough to accept `mjs` files
https://github.com/YunoHost-Apps/nextcloud_ynh/blob/96e835d9081730f09b2545680943c5790fce114f/conf/nginx.conf#L140

Comment (rodinux) [reply conf/nginx.conf:8]: I was also perhaps because of this discussion just before merge to 28
https://github.com/YunoHost-Apps/nextcloud_ynh/pull/622

Comment (rodinux) [reply conf/nginx.conf:8]: You are perhaps rigth, seems is better keep it reading the link to the official documentation https://docs.nextcloud.com/server/latest/admin_manual/installation/nginx.html#nextcloud-in-a-subdir-of-the-nginx-webroot

File: doc/PRE_UPGRADE.md [hunk 2]
```diff
@@ -1,3 +1,2 @@
 If you are upgrading to a new major version of Nextcloud, please make sure that your Nextcloud apps are up to date from Nextcloud's administration panel beforehand.
 
-Additionally, if you installed specific Nextcloud apps, we recommend making sure that they are compatible with the new major version. YunoHost will attempt to check this automatically at the very beginning of the upgrade, but a manual check doesn't hurt either. For Nextcloud 28, this forum thread might be helpful : <https://help.nextcloud.com/t/apps-not-compatible-with-nc-28/176234>.
```
Comment (alexAubin) [inline doc/PRE_UPGRADE.md:3]: Not sure why remove this recommendation ... It only mentions Nextcloud 28 as an example

File: conf/nginx.conf (hunk 1 above)
Comment (rodinux) [reply conf/nginx.conf]: I keep it, ok

**Comment by rodinux** [comment]
!testme

File: doc/PRE_UPGRADE.md [hunk 3]
```diff
@@ -1,3 +1,4 @@
 If you are upgrading to a new major version of Nextcloud, please make sure that your Nextcloud apps are up to date from Nextcloud's administration panel beforehand.
 
 Additionally, if you installed specific Nextcloud apps, we recommend making sure that they are compatible with the new major version. YunoHost will attempt to check this automatically at the very beginning of the upgrade, but a manual check doesn't hurt either. For Nextcloud 28, this forum thread might be helpful : <https://help.nextcloud.com/t/apps-not-compatible-with-nc-28/176234>.
+
```
Comment (alexAubin) [inline doc/PRE_UPGRADE.md:4]:
```suggestion
```

Unecessary new line ... Honestly at this point the history should be rewritten or squashed, half the commits were cancelled by other commits ...

Comment (OniriCorpe) [reply doc/PRE_UPGRADE.md:4]: (final empty lines in md files are added by some markdown formaters, because it seems to be standardized like thant)

**Review by ericgaspar — APPROVED** [review]

**Pull request merged**